omniStack11-frontend/src/app/app.service.ts

```diff
@@ -1,25 +1,32 @@
-import {API} from './app.api'
-import {HttpClient } from '@angular/common/http';
+import { NewCase } from './new-case/new-case.model';
+import { API } from './app.api'
+import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 
 import { Observable } from 'rxjs';
 
 import { Cases } from './cases/cases.module';
 //import { Logon } from './logon/logon.model';
 
-@Injectable()
 
+@Injectable()
 export class Services {
 
-    constructor(private http: HttpClient ){}
+    constructor(private http: HttpClient) { }
 
-    incidentsByOng():  Observable<Cases[]> {
-        return this.http.get<Cases[]>(`${API}/perfil`,{
+    incidentsByOng(): Observable<Cases[]> {
+        return this.http.get<Cases[]>(`${API}/perfil`, {
             headers: {
                 Authorization: '2a057067',
             }
         })
     }
 
-
+    newIncidentsByOng(newCase: NewCase): Observable<any> {
+        return this.http.post(`${API}/incidents`, newCase, {
+            headers: {
+                Authorization: '2a057067',
```

Comment:
Tu precisa passar um authorization nessa rota?
Se sim, tu deveria pegar o token do usuario que está logado.

Reply:
Sim, to passando fixo pq preciso pesquisar como faz.